src/libs/OptionsListUtils.js

```diff
@@ -424,42 +441,33 @@ function getOptions(reports, personalDetails, activeReportID, {
     selectedOptions = [],
     maxRecentReportsToShow = 0,
     excludeLogins = [],
-    excludeChatRooms = false,
     includeMultipleParticipantReports = false,
     includePersonalDetails = false,
     includeRecentReports = false,
-    prioritizePinnedReports = false,
     prioritizeDefaultRoomsInSearch = false,
 
     // When sortByReportTypeInSearch flag is true, recentReports will include the personalDetails options as well.
     sortByReportTypeInSearch = false,
     sortByLastMessageTimestamp = true,
     searchValue = '',
     showChatPreviewLine = false,
-    showReportsWithNoComments = false,
-    hideReadReports = false,
-    sortByAlphaAsc = false,
     sortPersonalDetailsByAlphaAsc = true,
     forcePolicyNamePreview = false,
-    prioritizeIOUDebts = false,
-    prioritizeReportsWithDraftComments = false,
 }) {
+    const isInGSDMode = priorityMode === CONST.PRIORITY_MODE.GSD;
     let recentReportOptions = [];
-    const pinnedReportOptions = [];
     let personalDetailsOptions = [];
-    const iouDebtReportOptions = [];
-    const draftReportOptions = [];
-
     const reportMapForLogins = {};
-    let sortProperty = sortByLastMessageTimestamp
+
+    // Filter out all the reports that shouldn't be displayed
+    const filteredReports = _.filter(reports, report => ReportUtils.shouldReportBeInOptionList(report, currentlyViewedReportID, isInGSDMode, currentUserLogin, iouReports, betas, policies));
+
+    // Now the filtered reports can be sorted
+    const sortProperty = sortByLastMessageTimestamp
         ? ['lastMessageTimestamp']
         : ['lastVisitedTimestamp'];
-    if (sortByAlphaAsc) {
-        sortProperty = ['reportName'];
-    }
-
-    const sortDirection = [sortByAlphaAsc ? 'asc' : 'desc'];
-    let orderedReports = lodashOrderBy(reports, sortProperty, sortDirection);
+    const sortDirection = ['desc'];
```

Comment:
NAB. It feels a bit weird to me having these constants defined as single-member arrays. Would it be better to make them strings and "arraying" them on the `lodashOrderBy` call?